### DISCUSSION TITLE
Show Hotspot Annotations on hover?

### DISCUSSION THREAD

**Discussion by pixlbndr** [google/model-viewer#3843] [Q&A]
Is it possible to have hotspot annotations appear when hovering over specific parts of the mesh?

**Comment by elalish** [comment]
Sure, you'd just need to call [`materialFromPoint`](https://modelviewer.dev/docs/#entrydocs-scenegraph-methods-materialFromPoint) in a mouse move listener. Author your GLB so each "specific part" you care about is a different material. Based on the material returned, change the display style of your hotspot element.
